### DISCUSSION TITLE
Question 90: Suppress word Bed [CM, WS]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1178] [Q&A]
Description:  For Vert Paleo can we suppress the addition of the word, “Bed” to the data entered into the field, “Bed” in Lilthostratrigraphy in Stratigraphy?

**Comment by rondlg** [comment]
Response: Please request this in a helpdesk ticket.

**Comment by rondlg** [comment]
FYI - @cmaves3 @magpiedin 

Can you add the ZD ticket number here please?

**Comment by magpiedin** [comment]
Zendesk [ticket #74368](https://fieldmuseum.zendesk.com/agent/tickets/74368)

(And related previous [ticket #72204](https://fieldmuseum.zendesk.com/agent/tickets/72204) with Geology EMu users ~ June 2024)
